source/expressions.tex

```diff
@@ -5708,20 +5708,20 @@
 \begin{itemize}
 \item If \tcode{P} evaluates to a null pointer value and
 \tcode{J} evaluates to 0, the result is a null pointer value.
-\item Otherwise, if \tcode{P} points to element $\mathtt{x[}i\mathtt{]}$
-of an array object \tcode{x} with $n$ elements,%
+\item Otherwise, if \tcode{P} points to an array element $i$
+of an array object \tcode{x} with $n$ elements\iref{dcl.array},%
 \footnote{An object that is not an array element is considered to belong to a
 single-element array for this purpose; see~\ref{expr.unary.op}.
 A pointer past the last element of an array \tcode{x} of $n$ elements
 is considered to be equivalent to a pointer to a hypothetical element
```

Comment:
Maybe insert "hypothetical <ins>array</ins> element $n$"?

Reply:
We do say "element" (without "array") later on, too. Maybe those should all be fixed.

Reply:
Fixed.

Reply:
Yes, indeed, that's what I was thinking, too. I think this is nice now.